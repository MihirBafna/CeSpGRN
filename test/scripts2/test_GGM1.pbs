#PBS -N test_GGM1
#PBS -q hive-himem
#PBS -l nodes=1:ppn=4
#PBS -l walltime=36:20:00
#PBS -j oe
#PBS -o test_GGM1.out
#PBS -m abe                            
#PBS -M zzhang834@gatech.edu  

cd $PBS_O_WORKDIR
module load anaconda3
conda activate myenv
python test_GGM1.py