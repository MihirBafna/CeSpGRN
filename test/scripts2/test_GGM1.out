---------------------------------------
Begin PBS Prologue Mon Oct  4 13:45:20 EDT 2021
Job ID:     7024029.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM1
Queue:      hive-himem
End PBS Prologue Mon Oct  4 13:45:20 EDT 2021
---------------------------------------
=>> PBS: job killed: walltime 130847 exceeded limit 130800
---------------------------------------
Begin PBS Epilogue Wed Oct  6 02:06:17 EDT 2021
Job ID:     7024029.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM1
Resources:  nodes=1:ppn=4,walltime=36:20:00,neednodes=1:ppn=4
Rsrc Used:  cput=145:01:49,vmem=354388100kb,walltime=36:20:57,mem=312891460kb,energy_used=0
Queue:      hive-himem
Nodes:     
atl1-1-01-018-35.pace.gatech.edu
End PBS Epilogue Wed Oct  6 02:06:17 EDT 2021
---------------------------------------
